experimental/c-cpp/manager.py

```diff
@@ -772,6 +852,96 @@ def evaluate_heuristic(test_dir, result_to_validate, fuzzer_intrinsics,
   shutil.copytree(fuzzer_gen_dir,
                   os.path.join(outdir, os.path.basename(fuzzer_gen_dir)))
 
+  # Create an OSS-Fuzz integration and ClusterFuzzLite integration
+  create_clean_oss_fuzz_from_success(
+      github_repo, os.path.join(outdir, os.path.basename(fuzzer_gen_dir)))
+  create_clean_clusterfuzz_lite_from_success(
+      github_repo, os.path.join(outdir, os.path.basename(fuzzer_gen_dir)))
+
+
+def create_clean_oss_fuzz_from_success(github_repo, success_dir):
+  """Converts a successful out dir into a working OSS-Fuzz project."""
+  oss_fuzz_folder = os.path.join(success_dir, 'oss-fuzz-project')
+  os.makedirs(oss_fuzz_folder)
+
+  # Project yaml
+  project_yaml = {
+      'homepage': github_repo,
+      'language': 'c++',
+      'primary_contact': 'add_your_email@here.com',
+      'main_repo': github_repo
+  }
+  with open(os.path.join(oss_fuzz_folder, 'project.yaml'), 'w') as project_out:
+    yaml.dump(project_yaml, project_out)
+
+  # Copy fuzzer
+  shutil.copy(os.path.join(success_dir, 'empty-fuzzer.cpp'),
+              os.path.join(oss_fuzz_folder, 'fuzzer.cpp'))
+
+  # Create Dockerfile
+  project_repo_dir = github_repo.split('/')[-1]
+  dockerfile = CLEAN_DOCKER % (github_repo, project_repo_dir, project_repo_dir)
+  with open(os.path.join(oss_fuzz_folder, 'Dockerfile'), 'w') as docker_out:
+    docker_out.write(dockerfile)
+
+  # Build file
+  with open(os.path.join(success_dir, 'build.sh'), 'r') as f:
+    build_content = f.read()
+  split_build_content = build_content.split("\n")
+
+  original_build_folder = split_build_content[1].split('/')[-1]
+
+  clean_build_content_lines = split_build_content[:1] + split_build_content[4:]
+  clean_build_content = '\n'.join(clean_build_content_lines).replace(
+      '/src/generated-fuzzer',
+      '$OUT/fuzzer').replace('empty-fuzzer', 'fuzzer').replace(
+          '/src/', '$SRC/').replace(original_build_folder, project_repo_dir)
+
+  with open(os.path.join(oss_fuzz_folder, 'build.sh'), 'w') as f:
+    f.write(clean_build_content)
+
+
+def create_clean_clusterfuzz_lite_from_success(github_repo, success_dir):
+  """Converts a successful out dir into a working ClusterFuzzLite project."""
+  cflite_folder = os.path.join(success_dir, 'clusterfuzz-lite-project')
+  os.makedirs(cflite_folder)
+
+  # Project yaml
+  project_yaml = {
+      'language': 'c++',
+  }
+  with open(os.path.join(cflite_folder, 'project.yaml'), 'w') as project_out:
+    yaml.dump(project_yaml, project_out)
+
+  # Copy fuzzer
+  shutil.copy(os.path.join(success_dir, 'empty-fuzzer.cpp'),
+              os.path.join(cflite_folder, 'fuzzer.cpp'))
+
+  # Create Dockerfile
+  project_repo_dir = github_repo.split('/')[-1]
+  dockerfile = CLEAN_DOCKER_CFLITE % (project_repo_dir, project_repo_dir)
```

Comment:
nit: can we use str.format here with named arguments here instead?

Reply:
Yes, done